Views/backend/connect/view/config/export/form.js

```diff
@@ -544,13 +546,36 @@ Ext.define('Shopware.apps.Connect.view.config.export.Form', {
                                                         );
                                                     }
                                                 });
+                                                Ext.getCmp('cbUseTriggers').setReadOnly(false);
```

Comment:
to prevent usage of Ext.getCmp you can do something like:

```
var useTriggersCheckbox = Ext.create('Ext.form.field.Checkbox', {
                            xtype: 'checkbox',
                            fieldLabel: me.snippets.activateDBTriggers,
                            helpText:  me.snippets.hintActivateDBTriggers,
                            name: 'useTriggers',
                            inputValue: 1,
                            uncheckedValue: 0,
                            labelWidth: me.defaults.labelWidth
                        });
```
Then id: 'cbUseTriggers' is not needed.
```
Ext.create('Ext.container.Container', {
                    columnWidth: 0.5,
                    layout: 'anchor',
                    border: false,
                    items: [useTriggersCheckbox]
                })
```

You're also able to call ```useTriggersCheckbox.setValue(0);``` instead ```Ext.getCmp('cbUseTriggers').setValue(0);```